tedana/resources/decision_trees/minimal.json

```diff
@@ -0,0 +1,231 @@
+                "ifFalse": "nochange",
+                "decide_comps": "all",
+                "op": ">",
+                "left": "countsigFS0",
+                "right": "countsigFT2"
+            },
+            "kwargs": {
+                "left2": "countsigFT2",
+                "op2": ">",
+                "right2": 0,
+                "log_extra_report": "",
+                "tag_ifTrue": "Unlikely BOLD"
+            }
+        },
+        {
+            "functionname": "calc_median",
+            "parameters": {
+                "decide_comps": "all",
+                "metric_name": "variance explained",
+                "median_label": "varex"
+            }
+        },
+        {
+            "functionname": "dec_left_op_right",
+            "parameters": {
+                "ifTrue": "rejected",
+                "ifFalse": "nochange",
+                "decide_comps": "all",
+                "op": ">",
+                "left": "dice_FS0",
+                "right": "dice_FT2"
+            },
+            "kwargs": {
+                "left2": "variance explained",
+                "op2": ">",
+                "right2": "median_varex",
+                "log_extra_report": "",
+                "tag_ifTrue": "Unlikely BOLD"
+            }
+        },
+        {
+            "functionname": "dec_left_op_right",
+            "parameters": {
+                "ifTrue": "rejected",
+                "ifFalse": "nochange",
+                "decide_comps": "all",
+                "op": ">",
+                "left": 0,
+                "right": "signal-noise_t"
+            },
+            "kwargs": {
+                "left2": "variance explained",
+                "op2": ">",
+                "right2": "median_varex",
+                "log_extra_report": "",
+                "tag_ifTrue": "Unlikely BOLD"
+            }
+        },
+        {
+            "functionname": "calc_kappa_elbow",
+            "parameters": {
+                "decide_comps": "all"
+            },
+            "kwargs": {
+                "log_extra_info": "",
+                "log_extra_report": ""
+            },
+            "_comment": ""
+        },
+        {
+            "functionname": "calc_rho_elbow",
+            "parameters": {
+                "decide_comps": "all"
+            },
+            "kwargs": {
+                "subset_decide_comps": "unclassified",
+                "rho_elbow_type": "liberal",
+                "log_extra_info": "",
+                "log_extra_report": ""
+            },
+            "_comment": ""
+        },
+        {
+            "functionname": "dec_left_op_right",
+            "parameters": {
+                "ifTrue": "provisionalaccept",
+                "ifFalse": "provisionalreject",
+                "decide_comps": "unclassified",
+                "op": ">=",
+                "left": "kappa",
+                "right": "kappa_elbow_kundu"
+            },
+            "kwargs": {
+                "log_extra_report": ""
```

Comment:
What part specifically? The `log_extra_report` fields?

Reply:
Yeah, I think one key thing tedana has always been missing is a sense of what each step is for and why it should be included. If we could have all of these messages filled in (at least for the minimal tree), it would really make the tree more transparent.